src/App.js

```diff
@@ -3,14 +3,17 @@ import AddTodoForm from './AddTodoForm';
 import TodoList from './TodoList';
 
 function App() {
-  const [newTodo, setNewTodo] = React.useState('');
+  const [todoList, setTodoList] = React.useState([]);
+
+  function addTodo(newTodo) {
+    setTodoList([...todoList, newTodo]);
+  }
 
   return (
     <>
       <h1>Todo List</h1>
-      <AddTodoForm onAddTodo={setNewTodo} />
-      <p>{newTodo}</p>
-      <TodoList />
+      <AddTodoForm onAddTodo={addTodo} />
```

Comment:
Nothing to change here, just my pet peeve regarding how they have you name stuff. I just think it's confusing to do it this way and `onAddTodo` is a bad function name because that's not what it does.

```suggestion
      <AddTodoForm addTodo={addTodo} />
```